transfervm/overlay/sbin/hotplug.stunnel

```diff
@@ -2,7 +2,7 @@
 
 sslversion=$(/usr/bin/xenstore-read "vm-data/transfer/ssl_version")
 if [ "$sslversion" == "" ]; then
-    sslversion="all"
+    sslversion="TLSv1.2"
```

Comment:
can you unconditionally set this to TLSv1.2? We are removing older protocols in another REQ.

Reply:
Same below about sslciphers, it should only use TLSv1.2

Reply:
I think here still remains one comment from Edwin. We can set sslversion and sslciphers to TLSv1.2 unconditionally.